mytk/base.py

```diff
@@ -57,6 +57,28 @@ def is_selected(self, value):
         else:
             self.widget.state(["!selected"])
 
+    @property
+    def width(self):
+        if self.widget is not None:
+            return self.widget['width']
+        return None
```

Comment:
this suggestion is linked to the suggestion on the width setter and to a comment on controls.py.

This would be done to allow less experienced developers to make mistakes and still get something that works as expected. The situation I have in mind is as follows :

```
slider1 = Slider()
slider1.width = 500
slider1.grid_into(app.window, ...
```

you would have a width of 200 even though you clearly said 500

```suggestion
        return self._width
```

Reply:
This was a new addition, not thought through really.  I added an exception if trying to set before widget is set.